tools/src/main/java/org/apache/kafka/trogdor/coordinator/CoordinatorClient.java

```diff
@@ -259,6 +267,11 @@ public static void main(String[] args) throws Exception {
                 readValue(res.getString("create_task"), CreateTaskRequest.class);
             client.createTask(req);
             System.out.printf("Sent CreateTaskRequest for task %s.", req.id());
+        } else if (res.getString("create_multiple_task") != null) {
```

Comment:
Perhaps `create_tasks` rather than `create_multiple_tasks`?  Also, did you add a command line option for this?